gdk4/src/display.rs

```diff
@@ -14,6 +14,9 @@ pub trait DisplayExtManual: 'static {
         group: i32,
     ) -> Option<(Key, i32, i32, ModifierType)>;
 
+    #[doc(alias = "gdk_display_get_setting")]
+    fn get_setting(&self, name: &str) -> Option<Value>;
```

Comment:
```suggestion
    fn get_setting(&self, name: &str) -> Option<glib::Value>;
```
either this or `use glib::Value` above & replace the other occurrences with `Value`